modules/component/types/extractValueTypeFromActionWithDeault.ts

```diff
@@ -0,0 +1,20 @@
+import {LActionValueForType} from './extractValueTypeFromAction'
+
+/**
+ * infers type of values from action set given action type's type
```

Comment:
what's the difference between this lambda and the one in `modules/component/types/extractValueTypeFromAction.ts`